lib/reporters/base.js

```diff
@@ -219,6 +219,11 @@ exports.list = function(failures) {
     stack = stack.replace(/^/gm, '  ');
 
     console.log(fmt, (i + 1), test.fullTitle(), msg, stack);
+
+    // [hack] Handle waterline's nested `originalError`
+    if (err.originalError && err.originalError instanceof Error) {
```

Comment:
Good q, I don't know. I could remote the 2nd condition, it would be harmless – is that what you're thinking?